redirect.json

```diff
@@ -134,6 +134,20 @@
       "blueprint_url": "https://github.com/home-assistant/core/blob/master/homeassistant/components/automation/blueprints/motion_light.yaml"
     }
   },
+  {
+    "redirect": "bluetooth_advertisement_monitor",
+    "name": "Bluetooth Advertisement Monitor",
```

Comment:
These names are very long, should we all rename them to "Browser"? So "Bluetooth Browser"

Reply:
Reduced to `Bluetooth Advertisements` and `Bluetooth Connections`